### PR TITLE
Introduce create job modal

### PR DESCRIPTION
**Pull request by orlandohohmeier**
---
ℹ️ This branch depends on features that will be introduced with #374, thus it will fail until the respective branch is merged.

---
create-job-modal
Introduce the a first version of the create job modal. This version only features the json editor. I'll add the missing features during the next iteration.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

File: src/js/components/modals/JobFormModal.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+  handleSubmit() {
+    let jsonDefinition = this.state.jsonDefinition;
+    ChronosStore.createJob(JSON.parse(jsonDefinition));
+    this.setState({
+      errorMessage: null,
+      jsonDefinition
+    });
+  }
+  getErrorMessage() {
```
Comment (kennyt) [inline src/js/components/modals/JobFormModal.js]: needa new line here

File: src/js/components/modals/JobFormModal.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
+import Ace from 'react-ace';
+import mixin from 'reactjs-mixin';
+import {Modal} from 'reactjs-components';
```
Comment (kennyt) [inline src/js/components/modals/JobFormModal.js]: :abc:

**Comment by kennyt** [comment]
after my nit, this looks good

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Pull request merged**